libs/parallelism/algorithms/include/hpx/algorithms/traits/projected.hpp

```diff
@@ -202,7 +232,11 @@ namespace hpx { namespace parallel { namespace traits {
         struct is_indirect_callable<ExPolicy, F, hpx::util::pack<Projected...>,
             typename std::enable_if<
                 hpx::util::all_of<is_projected_indirect<Projected>...>::value &&
+#if defined(HPX_HAVE_CXX20_EXPERIMENTAL_SIMD)
+                (!hpx::is_simdpack_execution_policy<ExPolicy>::value ||
+#else
                 (!hpx::is_vectorpack_execution_policy<ExPolicy>::value ||
+#endif
```

Comment:
I'd suggest to either specialize `is_vectorpack_execution_policy` for the `simd` types or to rename it to `is_simdpack_execution_policy` for Vc as well. Specializing this trait is preferable to littering our code with `#ifdef`s.